src/bots/ama/parser.js

```diff
@@ -0,0 +1,72 @@
+import client from '../../client';
+
+const parsePairs = { date: dateParser, participants: participantParser };
+
+/**
+ * Parses dictionary and returns trimmed data with proper conventions for Event schema
+ * @param {Object.<string, string>} dict - dictionary to be parsed
```

Comment:
not sure if we should be calling things dictionaries because that's very python, should prob call it object or map... but also not super important

Reply:
right I forgot JS they're objects... It's all too confusing

Reply:
could just do a find and replace on `dictionary` and `dict` haha